package-configuration/linux-cli/get-cloudify.py

```diff
@@ -425,9 +438,12 @@ def main():
     installer = CloudifyInstaller(args)
     installer.execute()
     if args.virtualenv:
-        prn('You can now run: "source {0}" to activate the Virtualenv.'.format(
-            os.path.join(args.virtualenv, 'bin/activate')))
+        lgr.info('You can now run: "source {0}" to activate '
+                 'the Virtualenv.'.format(
+                     os.path.join(args.virtualenv, 'bin/activate')))
+
 
+lgr = init_logger('get-cloudify')
```

Comment:
regarding `__name__`
that's the common python practice:
`The logger name hierarchy is analogous to the Python package hierarchy, and identical to it if you organise your loggers on a per-module basis using the recommended construction logging.getLogger(__name__). That’s because in a module, __name__ is the module’s name in the Python package namespace.`
from: https://docs.python.org/2/library/logging.html